modules/local/utils.nf

```diff
@@ -116,3 +116,20 @@ process CHANGE_UNDERSCORE_TO_DOT {
     """
 }
 
+/*
+ * clean reads, change . to _ from contigs
+*/
+process PUBLISH_CLEANED_PROK_BINS {
+
+    tag "${bins}"
+
+    input:
+    path bins
+
+    output:
+    path "*.fa"
+
+    script:
+    """
```

Comment:
![image](https://github.com/EBI-Metagenomics/genomes-generation/assets/1123897/6a65a886-27e8-4b15-be5a-e5095f2ef384)

Reply:
I don't know of a better solution for this.. (to publish code from within a workflow)